File: exercise12/ComputingSimpleInterest.java
package exercise12;

import java.util.Scanner;

public class ComputingSimpleInterest{
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Enter the principal: ");
        int principal = scanner.nextInt();

        System.out.print("Enter the rate of interest: ");
        double rateOfInterest = scanner.nextDouble();

        System.out.print("Enter the number of years: ");
        int years = scanner.nextInt();

        scanner.close();


        double amount = principal * (1 + (rateOfInterest / 100) * years);
    
        System.out.printf("After %d years at %.2f%%, the investment will be worth €%.2f",
            years, rateOfInterest, amount);
    }
}

File: 2. Calculations/exercise12/python/computing_simple_interest.py
# Exercise 12 - Computing Simple Interest

# Get input from user
principal = input('Enter the principal: ')
rate_of_interest = input('Enter the rate of interest: ')
years = input('Enter the number of years: ')

# Convert input to correct type
principal = int(principal)
rate_of_interest = float(rate_of_interest)
years = int(years)

# Perform Calculations
amount = principal * (1 + (rate_of_interest / 100) * years)

# Format output
output_text = f'After {years} years at {rate_of_interest}%, the investment' \
              f' will be worth €{amount:.2f}'

print(output_text)
